src/routes/handlers/director.handler.ts

```diff
@@ -0,0 +1,18 @@
+import * as express from 'express';
```

Comment:
I would argue that you don't need the `.handler` in the file name as it's implicit (almost explicit) in the file path `src/routes/handlers/director.ts`.

Should we unit test this?

Reply:
Or maybe comment that it is not unit tested because it's covered by the Cypress tests?

Reply:
Let's not argue! I don't mind either way. 
Was thinking about the unit tests, would be good, what framework?